WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -99,12 +101,15 @@ public class EditPostSettingsFragment extends Fragment {
 
     private SiteSettingsInterface mSiteSettings;
 
-    private LinearLayout mCategoriesContainer;
+    private LinearLayout mCategoriesTagsContainer;
```

Comment:
I think this refactor might be a risky move because the `Repository` exposes `PostModel` fields directly and that underlying `post` is being updated from more than one place (i.e.: `EditPostActivity`, `EditPostSettingsFragment`, `PublishSettingsViewModel`, and more) so ensuring proper data propagation might be complex and very likely to introduce bugs with the time I have to work on this. :sweat_smile:

Also, the `EditPostPublishSettingsViewModel` is used in a few other Fragments as well and it looks like it is specific to Publish Settings at the moment, so it would get a hit on single responsibility if I were to reuse that.

Unfortunately, I think it's safer to not do the refactor right now, although it would be a nice improvement. Does that sound good?